service/src/test/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelperTest.java

```diff
@@ -105,13 +95,13 @@ void subjectLineShouldReturnFirstRespondentElementAlwaysWhenMultipleRespondentsG
             .build();
 
         String expectedSubjectLine = "FamilyManCaseNumber-With-Empty-Lastname";
-        String subjectLine = helper.buildSubjectLine(caseData);
+        String subjectLine = buildSubjectLine(caseData.getFamilyManCaseNumber(), caseData.getRespondents1());
         assertThat(subjectLine).isEqualTo(expectedSubjectLine);
     }
 
     @Test
     void subjectLineShouldBeSuffixedWithHearingDate() {
-        final LocalDateTime dateInTenMonths = time.now().plusMonths(10);
+        final LocalDateTime dateInTenMonths = LocalDateTime.now().plusMonths(10);
```

Comment:
A slightly better version of this I think would remove the dependancy of this line
```
            + formatLocalDateTimeBaseUsingFormat(dateInTenMonths, "d MMM yyyy");

```
We could have a know date and then fully manually construct the expected subject line instead of relying on the helper to format it correctly.

Reply:
yup good idea :)